src/components/social-media/TweetButton.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+  export let text = "The latest from Cal-Adapt";
+  export let url = "https://cal-adapt.org";
+  export let hashtags = [];
+  export let via = "cal_adapt";
+  export let size = "large";
+</script>
+
+<div>
+  <a
+    class="twitter-share-button"
+    href="https://twitter.com/intent/tweet"
+    data-dnt="true"
+    data-text="{text}"
+    data-url="{url}"
+    data-hashtags="{Array.isArray(hashtags) && hashtags.length
+      ? hashtags.join(',')
+      : ''}"
+    data-via="{via}"
+    data-size="{size}"
+  >
+    Tweet</a
```

Comment:
Can we add the Twitter icon here? Using the `Icon Button` or `Link with Icon` component

Reply:
@mukhtyar it should look like this:
![image](https://user-images.githubusercontent.com/161748/132391877-52c5c2db-9b2c-48c7-b305-9727a995251b.png)

but for some reason the Twitter JS isn't working as expected so it just shows the word "Tweet" which I think is what you're seeing? If you try refreshing the page then you should see the Tweet button with the Twitter logo. 

I'll dig into it and see if I make it work, I might have to put the Twitter JS code in the component's `onMount` lifecycle.

Reply:
Got it. Still see just the word `Tweet` after refreshing the preview and not the button.

Reply:
I ended up rewriting the component and ditching the Twitter JS which was supposed to style the button. I think it looks better now if you want to check it locally (I haven't updated the Netlify preview deploy).